README.md

```diff
@@ -44,14 +44,21 @@ typo3_console/Scripts/typo3cms.bat to the TYPO3 root directory and set the locat
 
 Don't forget to activate the extension in the extension manager before you start using the command line tool.
 
-**TODO**
+#### linux shell installation
+make sure **"_cli_lowlevel"** user is a valid backenduser with any password**
+
+```
+cd typo3conf/ext/
+git clone https://github.com/helhum/typo3_console.git
+cp typo3_console/Scripts/typo3cms ../../
+cd ../../
```

Comment:
Why not simplify these 4 steps to 2 like this:

    git clone https://github.com/helhum/typo3_console.git typo3conf/ext/typo3_console
    cp typo3conf/ext/typo3_console/Scripts/typo3cms .